rxconf/rxconf.py

```diff
@@ -62,6 +74,15 @@ async def from_env(
     ) -> "AsyncMetaRxConf":
         pass
 
+    def _get_hash(self) -> bool:
+        return self._config.get_hash()
+
+    def __eq__(self, other) -> bool:
+        return self._get_hash() == other._get_hash()
```

Comment:
Encapsulation is broken, let's make get_hash public, because user doesn't use it directly and we don't complicate user experience.